docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -372,25 +370,25 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 > External access type to interactive jobs. Options are:
 >
->  * `nodeport`
->  * `external-url`
+> * `nodeport`
+> * `external-url`
 >
-> This flag can be used both in the standard format which uses command flag for all the options.
+> This flag now supports more than one `service-type`. Multiple service types are supported in CSV style using multiple instances of the same option and commas to separate the values for them. You can also add the optional parameter `custom-url` to the command.
```

Comment:
custom-url is only valid for external-url type! the comment you added doesn't explain it